### ISSUE TITLE
Button/link on paper page to open an issue about that paper?

### ISSUE THREAD

**Issue by nschneid** [acl-org/acl-anthology#2712]
If it's possible to do this and autofill the paper ID, it would make it easier to report corrections.

**Comment by mbollmann** [comment]
Yes, I had planned to do this already in #1826, but it fell off the radar. It will be good to have this issue as a reminder :)

**Comment by weissenh** [comment]
I think we have this now, so closing (also closed #524 as a very similar issue): 
the fix data button works and reports anthology id too. The GitHub action to automatically link and show paper thumbnail is super helpful too!

The only points I am aware of that might not be working is doing this on mobile (do we have an issue for this? was this a guess or a reproducible observation?) and when the data submitted to GitHub is too large ( issue 5303 )

**Comment by nschneid** [comment]
re: mobile, see discussion in #6518
